packages/expo-barcode-scanner/src/BarCodeScanner.tsx

```diff
@@ -23,6 +23,12 @@ export type BarCodeEventCallbackArguments = {
 
 export type BarCodeScannedCallback = (params: BarCodeEvent) => void;
 
+export type PermissionsRespone = {
+  status: "undetermined" | "granted" | "denied";
```

Comment:
We can go with sth like this: ```status: PermissionsStatus | "undetermined" | "granted" | "denied";``` for the time being, to ensure backward compatibility.